adam/curriculum_to_html.py

```diff
@@ -97,30 +98,35 @@
     "<li>Many objects also have associated Geons, which describe their shape "
     "according to Biederman's visual perception theory (see deliverable docs for a citation).</li>"
 )
+STR_TO_CURRICULUM = {
+    "phase1": GAILA_PHASE_1_CURRICULUM,
+    "prepositions": make_prepositions_curriculum(),
+}
 
 
 def main(params: Parameters) -> None:
     root_output_directory = params.creatable_directory("output_directory")
-    phase1_curriculum_dir = root_output_directory / "gaila-phase-1"
+    curriculum_string = params.optional_string(
+        "curriculum", valid_options=["phase1", "prepositions"], default="phase1"
```

Comment:
nit: you could do `valid_options=STR_TO_CURRICULUM.keys()` and then you don't need to worry about keeping them in sync.